examples/cloud-operations/network-dashboard/variables.tf

```diff
@@ -22,8 +22,14 @@ variable "billing_account" {
   description = "The ID of the billing account to associate this project with"
 }
 
+variable "monitoring_project_id" {
+  description = "Monitoring project where the dashboard will be created and the solution deployed; a project will be created if set to empty string"
+  default     = ""
+}
+
 variable "prefix" {
-  description = "Customer name to use as prefix for resources' naming"
+  description = "Customer name to use as prefix for monitoring project"
+  default     = ""
```

Comment:
I am in favor of having no default here so that each user sets their own prefix to get unique resources. What do you think?

Reply:
Agree but in that case even when you don't use the prefix because you don't create a new project you still have to fill the prefix. That's why I set a default of empty

Reply:
Got it.